README.md

```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
```

Comment:
Basically.. if we call the maintainers out that they have different expectations than the community for the call.. then we need to call out who the exclusions are for..